src/Microsoft.Health.Fhir.Core/Features/Persistence/BundleProcessingStatus.cs

```diff
@@ -0,0 +1,13 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Core.Features.Persistence
+{
+    public enum BundleProcessingStatus
+    {
+        SUCCEEDED,
```

Comment:
nit: Enums in this code base aren't all capitals.

Reply:
This enum will not be reuired as i will be using operationoutcome. I will  delete this file in my next update.